observations/obs_converters/NCEP/prep_bufr/prep_bufr.rst

```diff
@@ -1,67 +1,367 @@
+   own directory.
+
+   For a single obs file per directory, you can rename each as *obs_seq.out*,
+   or you can include the timestamp in the filename, e.g. *obs_seq.2017042706.out*.
+
+-  The observation types within the file should look like:
+
+   ::
+
+      obs_sequence
+      obs_type_definitions
+          19
+           1 RADIOSONDE_U_WIND_COMPONENT
+           2 RADIOSONDE_V_WIND_COMPONENT
+           5 RADIOSONDE_TEMPERATURE
+           6 RADIOSONDE_SPECIFIC_HUMIDITY
+          12 AIRCRAFT_U_WIND_COMPONENT
+          13 AIRCRAFT_V_WIND_COMPONENT
+          14 AIRCRAFT_TEMPERATURE
+          16 ACARS_U_WIND_COMPONENT
+          17 ACARS_V_WIND_COMPONENT
+          18 ACARS_TEMPERATURE
+          20 MARINE_SFC_U_WIND_COMPONENT
+          21 MARINE_SFC_V_WIND_COMPONENT
+          22 MARINE_SFC_TEMPERATURE
+          23 MARINE_SFC_SPECIFIC_HUMIDITY
+          30 SAT_U_WIND_COMPONENT
+          31 SAT_V_WIND_COMPONENT
+          40 RADIOSONDE_SURFACE_ALTIMETER
+          42 MARINE_SFC_ALTIMETER
+          43 LAND_SFC_ALTIMETER
+     num_copies:            1  num_qc:            1
+     num_obs:        68107  max_num_obs:        68107
+ 
+-  Some models include a preprocessing program to do additional processing
+   of the observations, for example, limiting obs to a sub-domain, superobbing spatially dense obs,
+   and increasing the error for near boundary observations.
+
+   For example, the WRF weather model includes the
+   :doc:`wrf_dart_obs_preprocess <../../../../models/wrf/WRF_DART_utilities/wrf_dart_obs_preprocess>`
+   program.  There may be no further processing needed for some models, and the observation sequence
+   file is ready to be used.
+
+
+**You have completed the Quickstart Instructions**. See the following sections for more details of the 
+prepbufr conversion package.
+
+
+
+Prepbufr Overview
+-----------------
+
+The prep_bufr package is external NCEP code and has not been completely incorporated into the DART architecture. It
+requires adaptation of the source codes and scripts to the computing environment where it will be run. It is not so
 robust that it can be controlled just with input parameters. It may not have the same levels of error detection and
 warning that the rest of DART has, so the user should very careful about checking the end product for correctness.
 
-Overview of what needs to be built and run
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-More detailed instructions follow, but this section describes a quick overview of what programs you will be building and
-running.
+Install Prepbufr package
+^^^^^^^^^^^^^^^^^^^^^^^^
 
-Building
-^^^^^^^^
-
-Running the install.sh script will build the library and main executable. You will probably have to edit this script to
-set which fortran compiler is available on your system.
+Running the ``install.sh`` script located within the ``~/observations/NCEP/prep_bufr`` directory will build the library
```

Comment:
home directory:
``~/observations/NCEP/prep_bufr``
should be 

$DART_DIR/observations/NCEP/prep_bufr